mezzanine_client/api.py

```diff
@@ -164,14 +164,16 @@ def get_post(self, item_id):
         """
         return self._get(['posts', int(item_id)])
 
-    def get_posts(self, offset=0, limit=10):
+    def get_posts(self, offset=0, limit=10, category="", date_min = ""):
         """
         Get published blog posts
         :param offset: pagination offset
         :param limit: pagination limit
+        :param category: the category which the post related to
```

Comment:
Please explicity state category name so it's clear that it's not the ID - i.e. change to `:param category: category name`